tensorflow_probability/tools/cmle_package.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import shutil
+import urllib2
+
+SOURCE_BASE = "https://raw.githubusercontent.com/tensorflow/probability/{branch}/tensorflow_probability/examples/{source_name}"
+BRANCH = 'r0.5'
+
+
+class CMLEPackage(object):
+    def __init__(self, source_name, requires=None, tfgfile_wrap=None):
```

Comment:
This file uses 4 spaces for indentation. There are other python files above which use 2 spaces for indentation. Let's make it consistent (I'd go with 4 spaces).